changelog.d/saf-1020.fixed

```diff
@@ -0,0 +1,5 @@
+Osemgrep only:
+
+When rules have metavariable-type, they don't show up in the SARIF output. This change fixes that.
```

Comment:
The style is a bit weird; We should be more direct like
Rules with metavariable-type do not show up in the SARIF output.
Dataflow traces are not shown anymore if you don't pass the --dataflow-traces.